### PR TITLE
[jsk_2016_baxter_apc] add recognition_in_hand and setup_torso

### LINKED ISSUE #1339: AttentionClipperが新ロボットモデルで動作しない
#1320 の原因を調べたところ、jsk_2015_05_baxter_apcのrecognition_in_hand.launchの中に、
```
  <node name="in_hand_clipper"
        pkg="nodelet" type="nodelet"
        args="load jsk_pcl/AttentionClipper $(arg MANAGER)"
        clear_params="true">
    <remap from="~input/points" to="$(arg INPUT_CLOUD)" />
    <rosparam>
      use_multiple_attention: true
      initial_pos_list: [[0.0, -0.1, 0.3], [0.0, -0.1, 0.3]]
      initial_rot_list: [[0, 0, 0], [0, 0, 0]] 
      dimensions: [[0.3, 0.2, 0.3], [0.3, 0.2, 0.3]]
      frame_id_list: [left_custom_vacuum_link_base, right_custom_vacuum_link_base]
      prefixes: [left_hand, right_hand]
    </rosparam>
  </node>
```
とあり、frame_id_listの中に入ってるリンクのうち、right_custom_vacuum_link_baseが新ロボットモデルに入っていないのが原因であるように思えた。しかし、認識プログラムについての知識が不足しており、わからない。
私のmilestoneではしばらく気にならないので、 @knorth55 にお願いしたい。

### PR DESCRIPTION
**Pull request by knorth55**
in_hand_clipper frame and bounding box is modified for 2016 gripper
Close #1339 

attention_clipper

### CODE REVIEW

File: jsk_2016_01_baxter_apc/launch/setup_torso.launch [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+  <include file="$(find jsk_2015_05_baxter_apc)/launch/segmentation_in_bin_gtol.launch">
+    <arg name="INPUT_IMAGE" value="/kinect2_torso/hd/image_color" />
+    <arg name="INPUT_DEPTH" value="/kinect2_torso/hd/image_depth_rect" />
+    <arg name="INPUT_CLOUD" value="/kinect2_torso/hd/points" />
+    <arg name="MANAGER" value="$(arg MANAGER)" />
+  </include>
+
+  <include file="$(find jsk_2015_05_baxter_apc)/launch/recognition_in_hand.launch">
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/launch/setup_torso.launch]: syntax error.

Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/setup_torso.launch]: thx

**Comment by wkentaro** [comment]
add test to check the launch file like:
https://github.com/start-jsk/jsk_apc/pull/1344

**Comment by wkentaro** [comment]
roslaunch-check does not support required arguments, and that's why it fails.
So could you please remove the latest commit? and then I will merge this.

**Comment by knorth55** [comment]
i removed it

**Pull request merged**